chainer_chemistry/datasets/molnet/molnet.py

```diff
@@ -129,6 +130,37 @@ def postprocess_label(label_list):
     return result
 
 
+def get_molnet_raw_dataframe(dataset_name):
+    """Downloads and caches MoleculeNet dataset without preprocessing.
+
+    Args:
+        dataset_name (str): MoleculeNet dataset name. If you want to know the
+            detail of MoleculeNet, please refer to
+            `official site <http://moleculenet.ai/datasets-1>`_
+            If you would like to know what dataset_name is available for
+            chainer_chemistry, please refer to `molnet_config.py`.
+    Returns (pandas.DataFrame or tuple):
+        DataFrame of dataset without any preprocessing. When the files of
+        dataset are seprated, this function returns multiple DataFrame.
+
+    """
+    if dataset_name not in molnet_default_config:
+        raise ValueError("We don't support {} dataset. Please choose from {}".
+                         format(dataset_name,
+                                list(molnet_default_config.keys())))
+    dataset_config = molnet_default_config[dataset_name]
+    if dataset_config['dataset_type'] == 'one_file_csv':
+        df = pandas.read_csv(get_molnet_filepath(dataset_name))
+        return df
+    elif dataset_config['dataset_type'] == 'separate_csv':
+        train_df = pandas.read_csv(get_molnet_filepath(dataset_name, 'train'))
+        valid_df = pandas.read_csv(get_molnet_filepath(dataset_name, 'valid'))
+        test_df = pandas.read_csv(get_molnet_filepath(dataset_name, 'test'))
+        return train_df, valid_df, test_df
+    else:
+        raise NotImplementedError
```

Comment:
Can you show message ?
'dataset_type={} is not supported'.format(dataset_config['dataset_type'])